server/models/Document.js

```diff
@@ -135,6 +137,11 @@ Document.associate = models => {
         { model: models.User, as: 'createdBy' },
         { model: models.User, as: 'updatedBy' },
       ],
+      where: {
+        publishedAt: {
+          [Sequelize.Op.ne]: null,
```

Comment:
re: scope, after adding this, it will become part of all the queries so it should be accounted in indeces